### PR TITLE
added github repos to projects section using a github token

### PR DESCRIPTION
**Pull request by AveryPratt**

### CODE REVIEW

**Comment by AveryPratt** [comment]
This took me a little over an hour after finishing up yesterdays assignment. It went pretty smoothly except for one bug where I listed my dependencies in the wrong order.

File: index.html [hunk 1]
```diff
@@ -49,6 +49,13 @@
         </div>
       </script>
     </section>
+    <section class="section" id="repos">
+      <h2>Repos</h2>
+      <script id="repo-template" type="text/x-handlebars-template">
+        <a href="{{html_url}}"><h3>{{name}}</h3></a>
+        <p>{{description}}</p>
```
Comment (patci-zz) [inline index.html:56]: Good!

File: js/models/repo.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+// TODO: create a githubToken.js file that we can use to generate our headers
+         // properly.
+        //  DONE
+  repos.requestRepos = function(callback) {
+    /* TODO: How would you like to fetch your repos? Someone say AJAX?!
+      Do not forget to call the callback! */
+    $.ajax({
+      type: 'GET',
+      headers: {'Authorization': 'token ' + githubToken,},
+      url: 'https://api.github.com/users/dbecker4130/repos',
```
Comment (patci-zz) [inline js/models/repo.js:16]: I recommend using your own GitHub name here, especially to get more practice writing out the code again.

File: js/models/repo.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    /* NOTE: This Model method filters the full repos collection based
+        on a particular attribute. For example, you could use this
+        to filter all repos that have a forks_count, stargazers_count,
+        or watchers_count. */
+    return repos.allRepos.filter(function(aRepo) {
+      return aRepo[myAttr];
+    });
+  };
+
+  module.repos = repos;
```
Comment (patci-zz) [inline js/models/repo.js:37]: Careful in this section. I'd avoid straight copy-paste from lab, and instead focus on typing more code out.

**Pull request merged**